2006Redone Server/src/redone/net/packets/impl/Bank10.java

```diff
@@ -28,14 +28,15 @@ public void processPacket(Client player, int packetType, int packetSize) {
 			player.getPlayerAssistant().useOperate(removeId);
 			break;
 		case 3900:
-			player.getShopAssistant().buyItem(removeId, removeSlot, 5);
+			player.getShopAssistant().buyItem(removeId, removeSlot, 10);

```

Comment:
Is there any reason to change the 5 option to 10?
I feel like this is just confusing the menu even more.